core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -155,6 +157,22 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        ctrl.getExplorationInteractions = function() {
+          $http.get(GET_INTERACTIONS_BY_ID_HANDLER_URL, {
+            params: {
+              exploration_id: ctrl.explorationId
+            }
+          })
+            .then(function(response) {
+              ctrl.explorationInteractionIds(
+                'Success! ' + response.data.interactions
```

Comment:
Maybe don't include 'success' since having results implies it succeeded.

Reply:
okay will remove